pkgs/tools/misc/yubikey-neo-manager/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, python27Packages, pyside, pycrypto
+, libykneomgr, yubikey-personalization, libu2f-host }:
+
```

Comment:
Please remove pyside, pycrypto here. They should already be in python27Packages. If they're not, we need to fix that.